src/tag.styles.ts

```diff
@@ -30,27 +30,35 @@ export default [
     .component {
       align-items: center;
       background: var(--glide-core-color-static-surface-container-secondary);
+      border: 1px solid var(--glide-core-color-interactive-stroke-primary);
       border-radius: var(--glide-core-rounding-base-radius-round);
-      color: var(--glide-core-color-static-text-default);
+      color: var(--glide-core-color-interactive-text-default);
       display: flex;
       font-family: var(--glide-core-typography-family-primary);
-      font-size: var(--glide-core-typography-size-body-small);
       font-weight: var(--glide-core-typography-weight-regular);
       justify-content: center;
       line-height: 1;
       max-inline-size: max-content;
-      min-block-size: var(--glide-core-spacing-base-md);
       opacity: 1;
-      padding: var(--glide-core-spacing-base-xxxs)
-        var(--glide-core-spacing-base-xs);
```

Comment:
Moved this and the ones above  to `.medium` given that's the size they apply to.